### PR TITLE
fix(tooltip): add max-width

### PR DESCRIPTION
**Pull request by JoomFX**
fixes https://github.com/telerik/kendo-themes/issues/816

### CODE REVIEW

File: packages/bootstrap/scss/_variables.scss [hunk 1]
```diff
@@ -1526,6 +1526,7 @@ $notification-error-border: tint($error, 9) !default;
 // Tooltip
 $tooltip-padding-y: $padding-y !default;
 $tooltip-padding-x: $padding-x !default;
+$tooltip-max-width: 250px !default;
```
Comment (TeyaVes) [inline packages/bootstrap/scss/_variables.scss]: For the Bootstrap theme, maybe we should keep the tooltip max-width of the Twitter Bootstrap, which is 200px

**Comment by TeyaVes** [comment]
The Kendo tooltip (in jQuery, as I see in Angular as well) has a setting for width and height and inline styles are added to the k-tooltip-content. Meaning, that if the width of the tooltip is set to equal 500px, but its parent has a max-width of 250px, the tooltip will overflow and won't look okay. So I am not sure if this is the best way to fix the issue here. Wouldn't a specified width fix the issue in IE? I am not able to open the stackblitz from the targeted issue in IE to check.

@elena-gancheva , @PreslavKozovski , what do you guys think?

**Comment by PreslavKozovski** [comment]
Adding max-width is not okay. This will break the jQuery Tooltip - https://docs.telerik.com/kendo-ui/api/javascript/ui/tooltip/configuration/width

**Comment by JoomFX** [comment]
Because of the exact issue @TeyaVes mentioned above, we needed to modify the Angular Tooltip component so it adds `max-width` (apart from the `width`) inline style.
After discussing the situation it seems that it will be best if we look for another solution and not merge this PR.

**Pull request closed without merging**